scan-triggers/aws-python-bucket-full-and-scheduled-scan/template.yaml

```
@@ -249,16 +249,25 @@ Resources:
     Properties:
       Code:
         ZipFile: |
-          const { S3Client, GetObjectCommand, PutObjectCommand } = require('@aws-sdk/client-s3');
-          const client = new S3Client();
+          const util = require('util')
+          const { PutObjectCommand, S3 } = require('@aws-sdk/client-s3');
+          const client = new S3({});
```

```suggestion
          const util = require('util')
          const { S3Client, GetObjectCommand, PutObjectCommand } = require('@aws-sdk/client-s3');
          const client = new S3Client({});
```
I don't think it's a good idea to import the whole S3 client here. Importing the aggregated version of `S3` will resulted in all the commands of s3 will be imported. Since only `getObject` and `putObject` is needed for this lambda function to do it's job. I think it will be better to follow the v3 SDK's modular concept and used `S3Client` as before.